largescale/run-million-collection-test.sh

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env bash
+# /usr/bin/env bash
```

Comment:
Is this shebang line change intentional?

Reply:
No! I totally missed that. Fixed.